src/code/audio_data.c

```diff
@@ -0,0 +1,674 @@
+    0.90643f,  0.907255f, 0.908081f, 0.908907f, 0.909734f, 0.910562f, 0.911391f, 0.91222f,  0.91305f,  0.913881f,
+    0.914713f, 0.915545f, 0.916379f, 0.917213f, 0.918047f, 0.918883f, 0.919719f, 0.920556f, 0.921394f, 0.922232f,
+    0.923072f, 0.923912f, 0.924752f, 0.925594f, 0.926436f, 0.927279f, 0.928123f, 0.928968f, 0.929813f, 0.93066f,
+    0.931507f, 0.932354f, 0.933203f, 0.934052f, 0.934902f, 0.935753f, 0.936604f, 0.937457f, 0.93831f,  0.939164f,
+    0.940019f, 0.940874f, 0.94173f,  0.942587f, 0.943445f, 0.944304f, 0.945163f, 0.946023f, 0.946884f, 0.947746f,
+    0.948608f, 0.949472f, 0.950336f, 0.951201f, 0.952066f, 0.952933f, 0.9538f,   0.954668f, 0.955537f, 0.956406f,
+    0.957277f, 0.958148f, 0.95902f,  0.959893f, 0.960766f, 0.961641f, 0.962516f, 0.963392f, 0.964268f, 0.965146f,
+    0.966024f, 0.966903f, 0.967783f, 0.968664f, 0.969546f, 0.970428f, 0.971311f, 0.972195f, 0.97308f,  0.973965f,
+    0.974852f, 0.975739f, 0.976627f, 0.977516f, 0.978405f, 0.979296f, 0.980187f, 0.981079f, 0.981972f, 0.982865f,
+    0.98376f,  0.984655f, 0.985551f, 0.986448f, 0.987346f, 0.988244f, 0.989144f, 0.990044f, 0.990945f, 0.991847f,
+    0.992749f, 0.993653f, 0.994557f, 0.995462f, 0.996368f, 0.997275f, 0.998182f, 0.999091f, 1.0f,      1.00091f,
+    1.001821f, 1.002733f, 1.003645f, 1.004559f, 1.005473f, 1.006388f, 1.007304f, 1.00822f,  1.009138f, 1.010056f,
+    1.010975f, 1.011896f, 1.012816f, 1.013738f, 1.014661f, 1.015584f, 1.016508f, 1.017433f, 1.018359f, 1.019286f,
+    1.020214f, 1.021142f, 1.022071f, 1.023002f, 1.023933f, 1.024864f, 1.025797f, 1.026731f, 1.027665f, 1.0286f,
+    1.029536f, 1.030473f, 1.031411f, 1.03235f,  1.033289f, 1.03423f,  1.035171f, 1.036113f, 1.037056f, 1.038f,
+    1.038944f, 1.03989f,  1.040836f, 1.041783f, 1.042731f, 1.04368f,  1.04463f,  1.045581f, 1.046532f, 1.047485f,
+    1.048438f, 1.049392f, 1.050347f, 1.051303f, 1.05226f,  1.053217f, 1.054176f, 1.055135f, 1.056095f, 1.057056f,
+    1.058018f, 1.058981f, 1.059945f, 1.06091f,  1.061875f, 1.062842f, 1.063809f, 1.064777f, 1.065746f, 1.066716f,
+    1.067687f, 1.068658f, 1.069631f, 1.070604f, 1.071578f, 1.072554f, 1.07353f,  1.074507f, 1.075485f, 1.076463f,
+    1.077443f, 1.078424f, 1.079405f, 1.080387f, 1.08137f,  1.082355f, 1.08334f,  1.084325f, 1.085312f, 1.0863f,
+    1.087289f, 1.088278f, 1.089268f, 1.09026f,  1.091252f, 1.092245f, 1.093239f, 1.094234f, 1.09523f,  1.096226f,
+    1.097224f, 1.098223f, 1.099222f, 1.100222f, 1.101224f, 1.102226f, 1.103229f, 1.104233f, 1.105238f, 1.106244f,
+    1.10725f,  1.108258f, 1.109267f, 1.110276f, 1.111287f, 1.112298f, 1.11331f,  1.114323f, 1.115337f, 1.116352f,
+    1.117368f, 1.118385f, 1.119403f, 1.120422f, 1.121441f, 1.122462f,
+};
+
+f32 gNoteFrequencies[] = {
+    /* 0x00 */ 0.105112f,   // NOTE_A0
+    /* 0x01 */ 0.111362f,   // NOTE_BFLAT0
+    /* 0x02 */ 0.117984f,   // NOTE_B0
+    /* 0x03 */ 0.125f,      // NOTE_C1
+    /* 0x04 */ 0.132433f,   // NOTE_DFLAT1
+    /* 0x05 */ 0.140308f,   // NOTE_D1
```

Comment:
The thing is, there's three possible ways to define note enums. One is from the start of this table i.e. `NOTE_A0 = 0x0`, another is from the lowest note possible `NOTE_BFLATNEG1 = 0x0`, (this table wraps around the bottom top for some reason), and the third is from middle C `NOTE_C4 = 0x0` which is defined as `1.0f`. So depending on the choice of enum, it may not correspond with the offset

Reply:
The thing is, there's three possible ways to define note enums. One is from the start of this table i.e. `NOTE_A0 = 0x0`, another is from the lowest note possible `NOTE_BFLATNEG1 = 0x0`, (this table wraps around the bottom top for some reason), and the third is from middle C `NOTE_C4 = 0x0` which is defined as `1.0f`. So depending on the choice of enum, it may not correspond with the offset

Reply:
I see. im assuming the enum comments here actually help with deciding which is best to use, while the offset comments arent really helping with that? or am i misunderstanding

Reply:
to clarify what i mean, for example the first entry corresponding to NOTE_A0 is what is important, not it being the 0th entry. at least thats what I gather, esp cause this is an array of frequencies

Reply:
It's mainly just there to make sure all information is available. So if you want to get at D1 Note, then you can quickly look at the table to know it's index 5. But it also depends on how low of a level you go, since this table is often accessed with an offset of -39 (which is middle C). So it could be accessed either relative to middle C or absolute.

Reply:
its probably my lack of understanding of the audio system keeping me from understanding, which is fine. if you think it should stay then that sounds good